apps/testingaccessibility/src/pages/index.tsx

```diff
@@ -24,7 +24,7 @@ const Home: React.FC<CommerceProps> = ({
 }) => {
   const {redeemableCoupon, RedeemDialogForCoupon} = useCoupon(couponFromCode)
   return (
-    <Layout className="bg-white">
+    <Layout className="bg-white" products={products}>
```

Comment:
not a big fan of layering state on a layout component like this and think that we need to take products/pricing up to the app as a context, will keep thinking on this one